### PR TITLE
Move run_script_in_container into mltsp sub-directory (fixes #80)

### LINKED ISSUE #80: Put `run_script_in_container` in a more sane location

### PR DESCRIPTION
**Pull request by acrellin**
Have to do a `sys.path.append` for the import to work now, which is why the script was originally placed where it was, though it's only run inside a Docker container so I think it's fine to do so.

### CODE REVIEW

**Comment by acrellin** [comment]
Fixed the Flask static data files issue

**Comment by acrellin** [comment]
Don't merge just yet, I found a minor bug I need to fix first

File: mltsp/Flask/flask_app.py [hunk 1]
```diff
@@ -109,6 +109,12 @@
     'XML', 'JSON'])
 
 
+# Data read from features directory
+@app.route('/features_data/<path:filename>')
```
Comment (stefanv) [inline mltsp/Flask/flask_app.py]: Isn't this a security hole?

**Comment by acrellin** [comment]
Ok, good to go

File: mltsp/run_script_in_container.py [hunk 2]
```diff
@@ -10,26 +11,16 @@
...
-    parser.add_argument('--featurize', action='store_true')
-    parser.add_argument('--build_model', action='store_true')
-    parser.add_argument('--predict', action='store_true')
     parser.add_argument('--tmp_dir', dest='tmp_dir', action='store', type=str)
     args = parser.parse_args()
 
     import sys
+    import os
     sys.path.append(args.tmp_dir)
+    sys.path.append(os.path.dirname(os.path.dirname(__file__)))
```
Comment (stefanv) [inline mltsp/run_script_in_container.py]: Instead of calling dirname twice, a slightly cleaner syntax is ``os.path.join(os.path.dirname(__file__), '..')``

Comment (stefanv) [inline mltsp/run_script_in_container.py]: Let's add a comment here as well to explain to future devs what's happening

File: mltsp/Flask/flask_app.py [hunk 3]
```diff
@@ -109,6 +109,24 @@
...
+
+
+# Data read from features directory
+@app.route('/features_data/<path:filename>')
+@stormpath.login_required
+def custom_static_features_data(filename):
+    if user_can_access_features_file(filename):
+        return send_from_directory(cfg.FEATURES_FOLDER, filename)
+    else:
+        return None
```
Comment (stefanv) [inline mltsp/Flask/flask_app.py]: Should return 403 Forbidden

**Comment by bnaul** [comment]
:+1: !

**Pull request merged**